src/library/LibraryWindow.vala

```diff
@@ -158,10 +158,11 @@ public class LibraryWindow : AppWindow {
         var import_menu_item = new Gtk.MenuItem ();
         import_menu_item.related_action = get_common_action ("CommonFileImport");
         import_menu_item.label = _("_Import From Folder…");
+        import_menu_item.use_underline = true;
 
         var preferences_menu_item = new Gtk.MenuItem ();
         preferences_menu_item.related_action = get_common_action ("CommonPreferences");
-        preferences_menu_item.label = _("_Preferences");
+        preferences_menu_item.label = _("Preferences");
```

Comment:
We should do the `user_underline` property here also to avoid breaking translations